main.py

```
@@ -0,0 +1,15 @@
+import yaml
+import logging
+
+from fraud_credit_cards.data_processor import DataProcessor
```

Ruff (0.8.2)

`fraud_credit_cards.data_processor.DataProcessor` imported but unused

Remove unused import: `fraud_credit_cards.data_processor.DataProcessor`

(F401)